lib/src/room.dart

```diff
@@ -282,6 +295,17 @@ class KurentoRoom extends ChangeNotifier implements Room {
 
         // Start the WebRTC signaling process.
         _connectTrack(track.id);
+
+        if (serviceRequestState == ServiceRequestState.queued) {
+          // HACK: If the Agent is sending audio and we still think we're queued, we're not receiving messages on the
+          // service request presence topic. Until we can figure out why this is happening, pretend we received a
+          // message and transition the service request status to started.
+          _log.shout('missed service request status message topic=$_serviceRequestPresenceTopic');
+          _agentName = '';
+          _serviceRequestState = ServiceRequestState.started;
+          _updateParticipantStatus();
+          notifyListeners();
+        }
```

Comment:
I'm hoping that changing the lifecycle of the MQTT client will eliminate the failures to subscribe to the service request presence topic, but in case it doesn't, this hack is here as a failsafe. If it turns out we don't need it, we'll remove it.